config/routes.rb

```diff
@@ -1,10 +1,13 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   get '/merchants/:id/items', to: 'merchant_items#index'
+  get '/merchants/:id/items/new', to: 'merchant_items#new'
```

Comment:
rather than handroll our routes maybe we should use the fancy `resources :some_name, only: [:index, :show]` format